sample.project.clj

```diff
@@ -7,14 +7,14 @@
 (defproject org.example/sample "1.0.0-SNAPSHOT"
   ; Your project must use Clojure 1.4 or above to support
   ; ClojureScript compilation.
-  :dependencies [[org.clojure/clojure "1.5.1"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
                  ; Your project should specify its own dependency on
                  ; ClojureScript
-                 [org.clojure/clojurescript "0.0-3211"
+                 [org.clojure/clojurescript "1.7.189"
```

Comment:
Thanks, changed. Not sure why GitHub shows the PR as up-to-date with the base branch now, it is not when you look at the commit.